php/build/alpine-3.7/Dockerfile

```diff
@@ -5,7 +5,10 @@ LABEL maintainer="joblocal GmbH <produktentwicklung@joblocal.de>"
 RUN apk add --update \
   curl \
   ghostscript \
+  git \
   imagemagick \
+  jq \
```

Comment:
I would not update our "older" versions. I would add the package to our most current images and rather migrate the services to the newer images. Otherwise the amount of maintenance will explode over time.

Reply:
Sry but the update of the alpine version is a separated topic in each project and we still want to add the deployment.